pephubclient/helpers.py

```diff
@@ -46,6 +46,15 @@ def send_request(
             params=params,
             json=json,
         )
+        if request_return.status_code == 401:
```

Comment:
Just confirming that the `401` is always returned for expired? Do we ever return 403? We shouldn't and I guess this is a pephub question. `403` is **unauthorized**, right? While `401` is unauthenticated?

Reply:
Cursory search on google seems to point to `401` being the correct code

Reply:
Thanks for checking:D